<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crack_detectionClass</class>
 <widget class="QMainWindow" name="crack_detectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>混凝土损伤检测系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>E:/file/LOGO集/微信图片_20201119165408.png</normaloff>E:/file/LOGO集/微信图片_20201119165408.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;测试记录：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>500</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>498</width>
       <height>168</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>500</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>498</width>
       <height>168</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>输入图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>测试结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>图片2.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>textEdit</zorder>
   <zorder>scrollArea</zorder>
   <zorder>scrollArea_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionopen_file"/>
    <addaction name="actionopen_folder"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="actionstart_testing"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>模型选择</string>
    </property>
    <addaction name="actionunet"/>
    <addaction name="actionunet_2"/>
    <addaction name="actionpspnet"/>
    <addaction name="actionupload_model"/>
    <addaction name="actioncustom_model"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen_file">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionopen_folder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actionstart_testing">
   <property name="text">
    <string>开始推理</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionunet">
   <property name="text">
    <string>unet</string>
   </property>
  </action>
  <action name="actionunet_2">
   <property name="text">
    <string>unet++</string>
   </property>
  </action>
  <action name="actionpspnet">
   <property name="text">
    <string>pspnet</string>
   </property>
  </action>
  <action name="actionupload_model">
   <property name="text">
    <string>ours model</string>
   </property>
  </action>
  <action name="actioncustom_model">
   <property name="text">
    <string>上传模型</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="crack_detection.qrc"/>
 </resources>
 <connections/>
</ui>
